source/auth/auth.rst

```diff
@@ -1175,31 +1175,39 @@ If AWS authentication fails for any reason, the cache MUST be cleared.
 .. note::
     Five minutes was chosen based on the AWS documentation for `IAM roles for EC2 <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html>`_ : "We make new credentials available at least five minutes before the expiration of the old credentials". The intent is to have some buffer between when the driver fetches the credentials and when the server verifies them.
 
+
 MONGODB-OIDC
 ~~~~~~~~~~~~
 
 :since: 7.0 Enterprise
 
-MONGODB-OIDC authenticates using an `OIDC
-<https://openid.net/specs/openid-connect-core-1_0.html>`_ access tokens. There
-are two workflows:
+MONGODB-OIDC authenticates using an `OpenID Connect (OIDC)
+<https://openid.net/specs/openid-connect-core-1_0.html>`_ access token. Most
+OIDC Identity Providers (*IdP*) support two identity federation flows:
+
+- Workload Identity Federation - Also called the "machine authentication flow",
+  this flow is intended to authorize access for software and services without
+  requiring human interaction.
+
+  TODO: Is there a 1:1 corresponding OIDC spec flow?
+
+- Workforce Identity Federation - Also called the "human authentication flow",
+  this flow is intended to authorize access for end users and typically requires
+  human interaction to complete (e.g. clicking a button in a browser).
 
-- Machine authentication workflow
-- Human authentication workflow
+  TODO: Is there a 1:1 corresponding OIDC spec flow?
```

Comment:
No, all authentication flows described in the OIDC spec are human authentication flows. I’d argue that it’s a bit confusing to refer to workload/workforce identity federation as “machine authentication flow”/“human authentication flow”, neither of these are really single auth flows in the sense of OIDC.

Reply:
Agreed, I was trying to relate the driver process to public OIDC documentation as much as possible, but I realized that was difficult because we're really satisfying a MongoDB auth mechanism that is based on OIDC, but is not an OIDC flow itself. I've updated the document to remove the explicit separation of "machine" and "human" flows, instead assuming the "machine" flow is the normal use case for "MONGODB-OIDC", and listing the "human" flow as a separate section that most drivers can ignore.